src/mats_l1_processing/instrument.py

```diff
@@ -16,7 +16,8 @@ class is used as a container for several CCDs. Finally a class for a generalized
 import pandas as pd
 import scipy.io
 from scipy.io import loadmat
-
+import sqlite3 as sqlite
```

Comment:
So from now on sqlite is a dependency for processing? What size are the files this reads (I suppose they will go in the instrument definition)?

Reply:
`sqlite` is not a problem as such, but I want to know if it is needed in the runtime, since it currently isn't. :)

700 MB, however, is going to be a breaking problem for the current method handling instrument data. Not unsolvable, but requiring a rethink of some concepts. This might be worth doing anyway – the current method is inelegant.